apps/wallet-mobile/src/features/Send/useCases/StartMultiTokenTx/StartMultiTokenTxScreen.tsx

```diff
@@ -154,14 +154,16 @@ const styles = StyleSheet.create({
   container: {
     flex: 1,
     backgroundColor: COLORS.WHITE,
-    paddingHorizontal: 16,
     paddingTop: 16,
   },
   flex: {
     flex: 1,
   },
   actions: {
-    paddingVertical: 16,
+    padding: 16,
+  },
+  scroll: {
```

Comment:
@SorinC6 this will push the scroll bar <-- 16, I think the padding should go inside the the `contentContainerStyle`

Reply:
@stackchain I have removed the container padding as well for this fix. Added horizontal padding for scroll and Actions buttons as are inside the container, added in the same as the other components have it.

Reply:
For fixing https://emurgo.atlassian.net/browse/YOMO-1144 - the scroll needs to have its own padding 
```
  container: {
    flex: 1,
    backgroundColor: COLORS.WHITE,
    paddingTop: 16,
  },
  flex: {
    flex: 1,
  },
  actions: {
    padding: 16,
  },
  scroll: {
    paddingHorizontal: 16,
  },
```